### PR TITLE
fix: add marcokumu

### LINKED ISSUE #1268: Okumu Mark

### PR DESCRIPTION
**Pull request by marcokumu**
resolves #1268 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by marcokumu** [comment]
Thanks.

File: profiles/marcokumu.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/90620737/181904436-f701dab8-5940-4572-951d-4c714cbca8ee.png",
+    "issueId": 1268,
+    "name": "Okumu Mark",
+    "username": "marcokumu"
+  }
```
Comment (Panquesito7) [inline profiles/marcokumu.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

File: profiles/marcokumu.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/90620737/181904436-f701dab8-5940-4572-951d-4c714cbca8ee.png",
+    "issueId": 1268,
+    "name": "Okumu Mark",
+    "username": "marcokumu"
```
Comment (Panquesito7) [inline profiles/marcokumu.json:5]: There seems to be a 4-space indentation. We use a 2-space indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/90620737/181904436-f701dab8-5940-4572-951d-4c714cbca8ee.png",
  "issueId": 1268,
  "name": "Okumu Mark",
  "username": "marcokumu"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by marcokumu** [comment]
The requested changes have been made.

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**